tests/site/xss_safe.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>XSS Safe</title>
+</head>
+<body>
+    <script>
+        window.escaped = "\u0027\u0027\u003B!\u002D\u002D\u0022\u003CXSS\u003E\u003D\u0026{()}";
+    </script>
+    <div id="fixture">
+        <div class="escaped">&#39;&#39;;!--&#34;&lt;XSS&gt;=&amp;{()}</div>
+        <input type="text" value="'';!--\&quot;<XSS>=&amp;{()}">
```

Comment:
@dianakhuang @efischer19 @cahrens Here's what I did:
1. Saved a local file with the contents of the HTML above, except I fully escape the contents of the value attribute (i.e. the value will match the contents of the `<div>`).
2. When I load this file in my browser, the text box correctly contains the value we expect.
3. When I view source in the browser, it displays the source as it was (i.e. the contents of my original file).

My current theory is that this code is written to work around a defect in the Accounts Setting page.  I'm not sure if there is still a defect there, or if it was fixed by the change around the jQuery val() call.  

Can you do what I did above in this test and see if we really do get a false positive?